_docs/feelgood.md

```diff
@@ -137,14 +137,26 @@ We used to the following resources as a guide for configuring S3 buckets to host
 
 ### API Gateway
 
-More info coming soon.
+There are multiple types of APIs that can be built using API Gateway, such as REST, HTTP, or WebSocket. This document will focus specifically on REST APIs. AWS' API Gateway documentation has a [helpful tutorial that covers creating a REST API with Lambda Integrations](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-getting-started-with-rest-apis.html). This was the approach taken for FeelGood.
+
+#### Create a Resource
+A resource is something that can be accessed through the REST API using CRUD operations. GCP has a good [explanation of REST APIs and resources](https://cloud.google.com/apis/design/resources) if you are unfamiliar with the concept or would like to refresh your memory. In API Gateway, the resource also represents your endpoint. For example, on a hypothetical website for a car dealership, the URL might be `www.cars.com/buycar`. The endpoint in this scenario is `/buycar`. Each endpoint has its own set of methods.
```

Comment:
THIS GOOGLE DESIGN GUIDE IS F*CKING AWESOME. I didn't even know it existed but it's beautifully written :')